core/third_party_libs/cuid/Cuid.php

```diff
@@ -0,0 +1,258 @@
+<?php
+
+namespace EndyJasmi;
+
+/**
+ * Cuid is a library to create collision resistant ids optimized for horizontal scaling and performance.
+ *
+ * @package EndyJasmi
+ */
+class Cuid
```

Comment:
Should we create an EE alias of this namespace/class and use that instead of consuming it directly? Just like our adapters approach for client side third-party libs.

Reply:
oh that's a great idea, going to create a PR for that, thnx